src/packet_sources/udp.rs

```diff
@@ -13,6 +14,16 @@ use crate::network::udp::{UdpHandler, UdpPacket};
 use crate::network::MAX_PACKET_SIZE;
 use crate::packet_sources::{PacketSourceConf, PacketSourceTask};
 
+pub fn remote_host_closed_conn(e: &Error) -> bool {
```

Comment:
We can turn this into
```rust
pub fn remote_host_closed_conn<T>(r: &Result<T, std::io::Error>) -> bool {
```
to simplify at the callsite?

Reply:
Ah yes, I just couldn't figure out how to handle a generic type